tests/installation/install_and_reboot.pm

```diff
@@ -36,6 +36,13 @@ sub handle_livecd_screenlock {
     assert_screen('yast-still-running', 120);
 }
 
+sub wait_countdown_stop {
+    my $stilltime = shift;
+    return wait_screen_change ( sub {
+        send_key 'alt-s';
+    }, $stilltime);
```

Comment:
I guess it should just be `return wait_screen_change { send_key 'alt-s' }, $stilltime`